### DISCUSSION TITLE
Help Breakup Long Lines

### DISCUSSION THREAD

**Discussion by rjv12345** [wxMaxima-developers/wxmaxima#1541] [General]
Hi
I am a very new user to wxMaxima 20.06.6
I get long expressions in my output
The expression goes far off to the right

How can I break an expression to show on screen without horizontal scrolling

Thanks
TC

**Comment by gunterkoenigsmann** [comment]
I did work on breaking long lines a lot, lately. Which operating system are you using? On Windows if you uninstall your maxima+wxMaxima an install the one from https://wolfgang.dautermann.at/maxima/nightlybuild/ instead your problem should be resolved. For debian-based systems there is https://code.launchpad.net/~maxima-developers/+archive/ubuntu/maxima-nightly, instead.

**Reply by rjv12345** [reply]
Hi,
Thank you for your quick reply.
I am using Win 10 O/S 
Before I uninstall and re-install the Wolfgang version, i have a question
Are there any unexpected , undocumented changes in the wolfgang version?
Will everything else be the same as wxMaxima 20.06.6 ?
Thanks
